controllers/installation_controller.go

```diff
@@ -104,7 +104,9 @@ var copyArtifactsJob = &batchv1.Job{
 							"/var/lib/embedded-cluster/bin/local-artifact-mirror pull binaries $INSTALLATION\n" +
 								"/var/lib/embedded-cluster/bin/local-artifact-mirror pull images $INSTALLATION\n" +
 								"/var/lib/embedded-cluster/bin/local-artifact-mirror pull helmcharts $INSTALLATION\n" +
-								"mv /var/lib/embedded-cluster/bin/k0s /var/lib/embedded-cluster/bin/k0s-upgrade",
+								"mv /var/lib/embedded-cluster/bin/k0s /var/lib/embedded-cluster/bin/k0s-upgrade\n" +
+								"rm /var/lib/embedded-cluster/images/images-amd64-*\n" +
```

Comment:
isn't this going to crash if no file exists under that path ? 

```
embedded-cluster: rm /var/does/no/exist
rm: cannot remove '/var/does/no/exist': No such file or directory
embedded-cluster: echo $?
1
embedded-cluster:
```

Reply:
let me check but I don't think so

Reply:
...I was wrong